oclmath/rounding_mode.cpp

```diff
@@ -220,6 +224,10 @@ void UnFlushToZero( void *p)
         #define _ARM_FE_FTZ     0x1000000
 		#define _ARM_FE_NFTZ    0x0
         #define _FPU_SETCW(cw) __asm__ ("VMSR FPSCR,%0" : :"ri" (cw))
+    #elif defined(__aarch64__)
+        #define _ARM_FE_FTZ     0x1000000
+		#define _ARM_FE_NFTZ    0x0
+        #define _FPU_SETCW(cw) __asm__ ("MSR FPCR,%0" : :"ri" (cw))
```

Comment:
```suggestion
        #define _ARM_FE_FTZ     0x1000000
    		#define _ARM_FE_NFTZ    0x0
        #define _FPU_SETCW(cw) __asm__ ("MSR FPCR,%0" : :"ri" (cw))
```

NOTE: your change invalidates the branch for `__arm__` case by removing
```c++
		static const int ftz_modes[ kRoundingModeCount ] = { _ARM_FE_FTZ, _ARM_FE_NFTZ };
		const int *f = ftz_modes;
		_FPU_SETCW( f[1] );
```

The same applies to `FlushToZero` function.